extension/sql/timescale_prometheus.sql

```diff
@@ -0,0 +1,62 @@
+
+CREATE OR REPLACE FUNCTION _prom_catalog.const_support(internal) RETURNS INTERNAL
```

Comment:
also maybe let's now give this function a better name. `const_arguments_initplan_support` ?